spec/requests/physical_servers_spec.rb

```diff
@@ -10,6 +10,155 @@
         expect_single_resource_query("ems_ref" => "A59D5B36821111E1A9F5E41F13ED4F6A")
       end
     end
+
+    context "without an appropriate role" do
+      it "fails to retrieve physical servers" do
```

Comment:
Nit, but the word "fails" is misleading. It succeeds in doing the right thing, which is to forbid access to the physical server